textdistance/libraries.py

```diff
@@ -97,6 +97,18 @@ def get_function(self):
                 # object constructor - the distance metric method is
                 # called dist_abs() (whereas dist() gives a normalised distance)
                 obj = getattr(module, self.func_name)().dist_abs
+            elif self.module_name in {'rapidfuzz.distance.Jaro', 'rapidfuzz.distance.JaroWinkler'}:
```

Comment:
Interesting 🤔 So, rapidfuzz considers two empty strings to have 0 similarity? Why? An empty string is equal to another empty string.

Reply:
Also, it does it only for Jaro but not for Hamming, if I understand correctly. Very interesting. Can we fix it upstream? Or there is anything in the original algorithm that says otherwise?

Reply:
Similar to `textdistance` I have the following algorithms
```
distance
similarity
normalized_similarity
normalized_distance
```

for two empty strings I handle this in the following way:
```
distance -> 0
similarity -> max - distance = 0 - 0
normalized_similarity -> 1.0 - normalized_distance -> 1.0
normalized_distance -> max / dist if max else 0 -> 0
```